docs/index.rst

```diff
@@ -0,0 +1,73 @@
+.. Images
+.. |thingspeak_1| image:: images/thingspeak_1.jpg
+.. |thingspeak_2| image:: images/thingspeak_2.jpg
+.. |thingspeak_3| image:: images/thingspeak_3.jpg
+.. |thingspeak_4| image:: images/thingspeak_4.jpg
+
+.. Links
+.. |ThingSpeak| raw:: html
+
+   <a href="https://thingspeak.com">ThingSpeak</a>
+
+ThingSpeak
+==========
+
+The *fledge-north-thingspeak* plugin provides a mechanism to |ThingSpeak|, allowing an easy route to send data from an Fledge environment into MATLAB.
+
+In order to send data to ThingSpeak you must first create a channel to receive it.
+
+  - Login to your |ThingSpeak| account
```

Comment:
We should add direct login link`https://thingspeak.com/login` instead of homepage. It's easy for beginners or any user